tools/csv-merger/csv-merger.go

```diff
@@ -160,6 +163,10 @@ func unmarshalCSV(filePath string) *csvv1.ClusterServiceVersion {
 				Name:  "ONBOARDING_SECRET_GENERATOR_IMAGE",
 				Value: *ocsContainerImage,
 			},
+			{
```

Comment:
Isn't ux a separate deployment, do we need this env?

Reply:
removing all reference of ux_backend_server_image related constants